src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -49,10 +53,10 @@ async function copyPageURL() {
 }
 ```
 
-### Paste: Reading text from the clipboard {: #reading-text }
+### Paste: reading text from the clipboard {: #reading-text }
 
 Much like copy, text can be read from the clipboard by calling
-`navigator.clipboard.readText()` and waiting for the returned Promise to
+`navigator.clipboard.readText()` and waiting for the returned promise to
```

Comment:
When referring generically to Promises, capitalize it. When referring to a specific instance, don't.

Reply:
Thanks for the style note.